README.md

```diff
@@ -129,6 +129,9 @@ directory as well as throughout our various open source projects.
 
 For complete API documentation, see [GoDoc](https://godoc.org/github.com/hashicorp/go-plugin).
 
+For an extensive tutorial on how to set up go-plugin and how it works, check
+out [Extensive go-plugin tutorial](docs/extensive-go-plugin-tutorial.md).
```

Comment:
@fairclothjm hey. :) 

Done, but I'm not a 100% this link will work. :D I can't see how you usually link between these... Any thoughts?

Reply:
My suggestion was to add a link in the `docs` directory README here: https://github.com/hashicorp/go-plugin/blob/master/docs/README.md. This commit adds it to the main repo README. Can you please add it in the `docs/README.md` table of contents?

I think we can just remove this from the main README. This will appear on https://pkg.go.dev/github.com/hashicorp/go-plugin#section-readme so I think your relative link will not work there. Maybe better to just remove this.

Reply:
ah, my bad :)